### DISCUSSION TITLE
metrics naming convention

### DISCUSSION THREAD

**Discussion by kareefardi** [The-OpenROAD-Project/OpenROAD#2928] [General]
Hello. I am wondering what is the metrics naming conventions. I am noticing that it is mostly `<flow stage/tool>__<metric>`. What I am unsure about is how to name the `<metric>` part. There is a mix usage of double underscore and single underscore as separators. For example there are `route__drc_errors`, `route__vias__singlecut` and `timing__drv__hold_violation_count`.

**Comment by maliberty** [comment]
The naming is described at https://github.com/ieee-ceda-datc/datc-rdf-Metrics4ML.  @ravi-varadarajan can help with questions.
